styleguide/_themes/derek/scss/components/_footer.scss

```diff
@@ -1,3 +1,15 @@
-.footer{
-  
-}
\ No newline at end of file
+.rug{
```

Comment:
@bighappyface You want the rug, footer menu and basement in their own files? Those two things have very few lines styling wise, I think it's a little less confusing to have those all in one file.